MendelaA/HW7/PT2/5.py

```diff
@@ -0,0 +1,16 @@
+"""
+You need to write a function that reverses the words in a given string.
+A word can also fit an empty string. If this is not clear enough, here are some examples:
+As the input may have trailing spaces, you will also need to ignore unneccesary whitespace.
+"""
+
```

Comment:
good job
look at this optimization
def reverse(st):
    return ' '.join(st.split()[::-1])